server/controllers/headers.js

```diff
@@ -0,0 +1,9 @@
+export const BggFilterPlayerCount = 'Bgg-Filter-Player-Count';
+export const BggFilterMinDuration = 'Bgg-Filter-Min-Duration';
+export const BggFilterMaxDuration = 'Bgg-Filter-Max-Duration';
+
+export function filterHeaders(requestHeaders) {
+    return Object.entries(requestHeaders)
+        .filter(([key, val]) => val !== undefined) //eslint-disable-line no-unused-vars

```

Comment:
FYI, if you want to respect the linting rule you can ignore the first param

```suggestion
        .filter(([ , val]) => val !== undefined) //eslint-disable-line no-unused-vars
```
It's up to you, which way you find clearer (I don't have a strong view), but I thought I'd throw the option out.
(the space is unnecessary but may make it harder to miss)

Reply:
I didn't know i could do that, but I prefer it!